plugins/extractit/plugin.spec.yaml

```
@@ -0,0 +1,292 @@
+plugin_spec_version: v2
+extension: plugin
+products: ["insightconnect"]
+name: extractit
+title: ExtractIt
+description: The ExtractIt plugin has a collection of actions used to extract various information from text. These include URLs, emails, and IPs
+version: 1.0.0
```

```suggestion
version: 2.1.1
```

This will need to be the next version after the current one (2.1.0) at https://extensions.rapid7.com/extension/extractit